# tls.connect does not send SNI by default, so the server otherwise
# selects its default certificate instead of the one for www.nodejs.org.
# https://nodejs.org/api/tls.html#tlsconnectoptions-callback
# Upstream: https://github.com/nodejs/node/pull/66835
diff --git a/test/internet/test-tls-add-ca-cert.js b/test/internet/test-tls-add-ca-cert.js
--- a/test/internet/test-tls-add-ca-cert.js
+++ b/test/internet/test-tls-add-ca-cert.js
@@ -27,5 +27,6 @@ const caCert = loadPEM('ca1-cert');
 const opts = {
   host: 'www.nodejs.org',
+  servername: 'www.nodejs.org',
   port: 443,
   rejectUnauthorized: true,
 };
